web/client/epics/__tests__/wfsquery-test.js

```diff
@@ -347,37 +377,51 @@ describe('wfsquery Epics', () => {
                 });
                 done();
             },
-            {
-                query: {
-                    data: {},
-                    featureTypes: [],
-                    typeName: 'layer1',
-                    url: '/dummy'
-                },
-                featuregrid: {
-                    timeSync: true,
-                    pagination: {
-                        size: 10
-                    },
-                    open: true,
-                    selectedLayer: "layer1",
-                    changes: [],
-                    mode: 'VIEW'
-                },
-                layers: {
-                    flat: flatLayers,
-                    layerMetadata: {
-                        expanded: false,
-                        maskLoading: false
-                    },
-                    settings: {
-                        expanded: false,
-                        node: null,
-                        nodeType: null,
-                        options: {}
-                    }
+            mockStore
+            );
+        });
+
+        it('featureTypeSelectedEpic', (done) => {
+            const wfsResults = require('../../test-resources/wfs/describe-pois.json');
+            mockAxios.onGet().reply(() => [200, wfsResults]);
+            testEpic(featureTypeSelectedEpic, 1, [
```

Comment:
```suggestion
            testEpic(featureTypeSelectedEpic, 2, [
```
You espect 2 actions before to run the test callback (4th arg)